collections/_pages/talks-and-presentations.html

```diff
@@ -132,12 +230,114 @@ <h1>An Introduction to Meshery (CNCF Webinar-on-Demand)</h1>
             </div>
           </div>
         </div>
-      </p>
+      
     </div>
     <div class="image" >
       <img src="{{ site.baseurl }}/images/events/cncf-webinar-meshery.png"
         alt="Service Mesh configuration Comparison" loading="lazy"/>
     </div>
+    
   </div>
+  
 </section>
-<script src="{{  site.baseurl }}/assets/js/modal.js"></script>
+<table>
+  <thead>
+      <tr>
+          <th class="topic">Topic</th>
+          <th>Resources</th>
+          <th>Presenter</th>
+      </tr>
+  </thead>
+  <tbody>
+    {% for talk in site.data.talks %}
+      {% if talk.type != 'tutorial' %}
+        <tr >
+            <td>{{talk.title}}</td>
+            <td class = "table-cell-style">
+              {% if talk.video %}
+                <svg class="recording-link" fill="#FE3C00" xmlns="http://www.w3.org/2000/svg"  viewBox="0 0 16 16" width="25px" height="25px"><path d="M 2.488281 3.011719 C 1.664063 3.011719 0.988281 3.6875 0.988281 4.511719 L 0.988281 11.511719 C 0.988281 12.335938 1.664063 13.011719 2.488281 13.011719 L 13.488281 13.011719 C 14.308594 13.011719 14.988281 12.335938 14.988281 11.511719 L 14.988281 4.511719 C 14.988281 3.6875 14.3125 3.011719 13.488281 3.011719 Z M 2.488281 4.011719 L 13.488281 4.011719 C 13.769531 4.011719 13.988281 4.230469 13.988281 4.511719 L 13.988281 11.511719 C 13.988281 11.792969 13.769531 12.011719 13.488281 12.011719 L 2.488281 12.011719 C 2.207031 12.011719 1.988281 11.792969 1.988281 11.511719 L 1.988281 4.511719 C 1.988281 4.230469 2.207031 4.011719 2.488281 4.011719 Z M 6 4.992188 L 6 11.011719 L 11 8 Z M 7 6.757813 L 9.0625 8 L 7 9.242188 Z"/></svg>
```

Comment:
Instead of mentioning the svg here, could you please put it in the images folder and call it here, also, we might want to re-think the icons being used. Everything else looks good though.

Reply:
done.

Reply:
can you suggest a different icon, I will update that too and if it's possible I also need suggestion regarding color of the icons.